website/docs/api/admin/user-admin.md

```diff
@@ -115,7 +115,17 @@ You can also search for users via the search API. It will preform a simple searc
 
 `POST https://unleash.host.com/api/admin/user-admin`
 
-Creates a new use with the given root role.
+Creates a new user with the given root role.
+
+**Payload properties**
+
+| Property name | Required | Description                                                                     | Example value(s)       |
+|---------------|----------|---------------------------------------------------------------------------------|------------------------|
+| `email`       | Yes      | The user's email address.                                                       | `"user@getunleash.io"` |
+| `name`        | Yes      | The user's name                                                                 | `"Some Name"`          |
+| `rootRole`    | Yes      | The role to assign to the user. Can be either the role's ID or its unique name. | `2`, `"Editor"`        |
+| `sendEmail`   | No       | Whether to send a registration email to the user or not. Defaults to `true`.    | `false`                |
```

Comment:
Is this information correct? What kind of email is sent out? What's the default value (or is it required)?

Reply:
If I remember correctly the email is the welcome email with a login link for setting the password for the user.

Reply:
I do believe it defaults to true (as in, it's not required).